doc/user_guide/installation/command_line_installation.rst

```diff
@@ -10,6 +10,13 @@ works with your interpreter. We recommend ``pip``:
 
    pip install pylint
 
+Or if you want to also check spelling with ``enchant`` (you might need to
+`install the enchant C library <https://pyenchant.github.io/pyenchant/install.html#installing-the-enchant-c-library>`_ ):
+
+.. code-block:: sh
+
+   pip install pylint[spelling]
+
 Use the newest Python interpreter if you can.
 
```

Comment:
```suggestion
```

This is such a weird tip. I think that is really something for the user to decide 😄

Reply:
It's in the context of supporting 3.5 to 3.9, you should use pylint with python 3.9 and set ``py-version`` to 3.5.

Reply:
Yeah, but shouldn't that be in the configuration section somewhere? This feels a bit out of place.

Reply:
We're talking about what interpreter to use during installation. Maybe we should group this sentence it in the paragraph just under this so it's clearer ? I don't know if most user maintain multiple interpreters though, so we could move this on the assumption that most user just use their current interpreter ?

Reply:
> I don't know if most user maintain multiple interpreters though, so we could move this on the assumption that most user just use their current interpreter ?

Yeah I think this is a fair assumption. If users use different interpreters they likely also know that it is better to use the most recent interpreter for their tools.